### PR TITLE
Develop > Blockchains > Custom Blockchains > Overview

### PR DESCRIPTION
**Pull request by dawnkelly09**
This PR:
- uses content from previously merged PR #92 to create this page
- updates the Introduction
- moves and updates paths for snippets and images
- moves a code block out to a snippet

### CODE REVIEW

**Review by CrackTheCode016 — APPROVED** [review]
lgtm besides that minor comment!

File: develop/blockchains/custom-blockchains/overview.md [hunk 1]
```diff
@@ -1,4 +1,86 @@
...
+This modular design allows developers to:
+
+- Rapidly prototype blockchain systems
+- Easily add or remove features by including or excluding pallets
+- Customize blockchain behavior without rebuilding core components
+- Leverage tested and optimized code from built-in pallets
+
+For more detailed information on implementing this process, refer to the following sections:
+
+- [Add a Pallet to Your Runtime](/develop/blockchains/custom-blockchains/add-existing-pallets.md){target=\_blank}
```
Comment (eshaben) [inline develop/blockchains/custom-blockchains/overview.md:85]:
```suggestion
- [Add a Pallet to Your Runtime](/develop/blockchains/custom-blockchains/add-existing-pallets/){target=\_blank}
```

File: develop/blockchains/custom-blockchains/overview.md [hunk 2]
```diff
@@ -1,4 +1,86 @@
...
+- [Create a Custom Pallet](/develop/blockchains/custom-blockchains/make-custom-pallet.md){target=\_blank}
```
Comment (eshaben) [inline develop/blockchains/custom-blockchains/overview.md:86]:
```suggestion
- [Create a Custom Pallet](/develop/blockchains/custom-blockchains/make-custom-pallet/){target=\_blank}
```

**Review by eshaben — COMMENTED** [review]
Noticed a couple link issues, but other than that it looks good

File: develop/blockchains/custom-blockchains/overview.md [hunk 3]
```diff
@@ -1,4 +1,86 @@
...
----
\ No newline at end of file
+description: Learn how Polkadot SDK’s FRAME framework simplifies blockchain development with modular pallets and support libraries for efficient runtime design.
+---
+
+# Overview
+
+## Introduction
+
+The runtime is the heart of any Polkadot SDK-based blockchain, handling the essential logic that governs state changes and transaction processing. With Polkadot SDK’s [FRAME (Framework for Runtime Aggregation of Modularized Entities)](/glossary/#frame-framework-for-runtime-aggregation-of-modularized-entities){target=\_bank}, developers gain access to a powerful suite of tools for building custom blockchain runtimes. FRAME offers a modular architecture, featuring reusable pallets and support libraries, to streamline development.
```
Comment (eshaben) [inline develop/blockchains/custom-blockchains/overview.md:10]:
```suggestion
The runtime is the heart of any Polkadot SDK-based blockchain, handling the essential logic that governs state changes and transaction processing. With Polkadot SDK’s [FRAME (Framework for Runtime Aggregation of Modularized Entities)](/polkadot-protocol/glossary/#frame-framework-for-runtime-aggregation-of-modularized-entities){target=\_bank}, developers gain access to a powerful suite of tools for building custom blockchain runtimes. FRAME offers a modular architecture, featuring reusable pallets and support libraries, to streamline development.
```

**Review by eshaben — COMMENTED** [review]
One last issue

**Review by eshaben — APPROVED** [review]

**Pull request merged**